MinimalMVVM/ViewModel/Presenter.cs

```diff
@@ -10,7 +10,7 @@ public class Presenter : ObservableObject
     {
         private readonly TextConverter _textConverter = new TextConverter(s => s.ToUpper());
         private string _someText;
-        private readonly ObservableCollection<string> _history = new ObservableCollection<string>();
+        public ObservableCollection<string> History { get; } = new ObservableCollection<string>();
```

Comment:
The problem with this is it forces more information to be leaked through the public API than is strictly necessary. An argument could be made that it is fine in this particular situation.

Reply:
Also, it would be more consistent if this was in the same position in the file as the original property, rather than the position of the backing field.